lib/matplotlib/contour.py

```diff
@@ -1532,7 +1542,7 @@ def _check_xyz(self, args, kwargs):
         Exception class (here and elsewhere).
         """
         x, y = args[:2]
-        self.ax._process_unit_info(xdata=x, ydata=y, kwargs=kwargs)
+        kwargs = self.ax._process_unit_info(xdata=x, ydata=y, kwargs=kwargs)
```

Comment:
I am :-1: on renaming it because it has had the current name for a long time and has always probably had this pop behavior.

In any case that change is outside of the scope if this PR.